### PR TITLE
API-3120: Hook Migration EmbeddedYoutubeVideo, MarkdownPage, Hero

### PR DESCRIPTION
**Pull request by TS000**
### Description
Moved components to folders and updated hooks. Jira ticket [here](https://vajira.max.gov/browse/API-3120).

- [ NA ] Tests added or updated for change
- [ NA ] Docs updated
- [ NA ] Accessibility concerns have been considered and addressed

### Requested feedback
All feedback is welcome.

### CODE REVIEW

File: src/Routes.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { getDeactivatedFlags } from './apiDefs/deprecated';
 import { getEnvFlags } from './apiDefs/env';
 import { getApiCategoryOrder, getApiDefinitions } from './apiDefs/query';
 import { APIDescription } from './apiDefs/schema';
-import MarkdownPage from './components/MarkdownPage';
+import { MarkdownPage } from './components/markdownPage/MarkdownPage';
```
Comment (mjlumetta) [inline src/Routes.tsx:9]:
```suggestion
import { MarkdownPage } from './components';
```

**Review by mjlumetta — COMMENTED** [review]
one carry-over comment but this is in good shape!

File: src/Routes.tsx (hunk 1 above)
Comment (mjlumetta) [reply src/Routes.tsx:9]: also: i think another option would be to inline `MarkdownPage` in this file if you'd prefer. it's only used here, and it's like 3 tags deep. could be nice to reduce the size of `src/components` by one directory.

**Review by mjlumetta — DISMISSED** [review]

**Review by anthony-kopczyk — APPROVED** [review]
Looks good to me!

**Comment by mjlumetta** [comment]
@TS000 can this one close? no diff with the base branch, looks like all those commits were in #531

**Comment by TS000** [comment]
> @TS000 can this one close? no diff with the base branch, looks like all those commits were in #531

I need some reviews. Other than that I think it should be gtg.

**Pull request closed without merging**